pages/studentAssessmentExam/studentAssessmentExam.ejs

```diff
@@ -31,23 +31,44 @@
             </strong>
           </p>
           <% } %>
-          <form id="confirm-form" name="confirm-form" method="POST">
+
+          <% if (locals.assessment.require_honor_code) { %>
+          <div class="card card-secondary mb-4 test-class-honor-code">
+            <ul class="list-group list-group-flush">
+              <li class="list-group-item py-2">I certify that I am <%= user.name %> and I am allowed to take this assessment in <%= course.short_name %> (<%= course_instance.short_name %>).</li>
+              <li class="list-group-item py-2">I pledge on my honor that I will not give or receive any unauthorized assistance on this assessment and that all work will be my own.</li>
+            </ul>
+
+            <div class="card-footer text-center border-top-0">
+              <span class="form-check d-inline">
+                <input type="checkbox" class="form-check-input" id="agree-certify">
+                <label class="form-check-label font-weight-bold" for="agree-certify">I agree to and certify the above.</label>
+              </span>
+            </div>
+          </div>
+          <% } %>
```

Comment:
I really like the way this looks, with the bold-and-centered checkbox statement. Very neat!